aospy/internal_names.py

```diff
@@ -42,8 +42,8 @@
 AVERAGE_T2_STR = 'average_T2'
 TIME_WEIGHTS_STR = 'time_weights'
 NV_STR = 'nv'
-AVG_START_DATE_STR = 'avg_start_date'
-AVG_END_DATE_STR = 'avg_end_date'
+AVG_START_DATE_STR = 'data_start_date'
+AVG_END_DATE_STR = 'data_end_date'
```

Comment:
@spencerahill I think these attributes could use a name change.  I'm not sure what the best option would be, but in this case, I think the repr of computed data is a little misleading.  E.g. this is a calculation that was done on years 3 through 6 on some idealized model output:
```
<xarray.Dataset>
Dimensions:               (lat: 64, lon: 128, pfull: 30)
Coordinates:
  * lon                   (lon) float64 0.0 2.812 5.625 8.438 11.25 14.06 ...
  * lat                   (lat) float64 -87.86 -85.1 -82.31 -79.53 -76.74 ...
  * pfull                 (pfull) float64 3.385 10.78 14.5 19.3 25.44 33.2 ...
    avg_start_date        datetime64[ns] 1678-01-01
    avg_end_date          datetime64[ns] 1684-01-01
    sfc_area              (lat, lon) float64 3.553e+09 3.553e+09 3.553e+09 ...
    land_mask             (lat, lon) float64 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 ...
Data variables:
    equiv_potential_temp  (pfull, lat, lon) float64 999.5 999.6 999.6 999.7 ...
```
These attributes make it seem as though the length of the time period the average was computed for was 6 years, when this is not the case.  These attributes describe the full length of the data available to the DataLoader (not what is ultimately subset).  I'm not super happy with the names I have here.  Do you have any better ideas?